coldfront/plugins/qumulo/utils/acl_allocations.py

```diff
@@ -37,45 +37,21 @@ def add_allocation_users(self, allocation: Allocation, wustlkeys: list):
             )
 
     @staticmethod
-    def add_user_to_access_allocation(username: str, allocation: Allocation):
+    def add_user_to_access_allocation(
```

Comment:
`add_member_to_access_allocation` ?

Reply:
Debatable, since it is adding an AllocationUser...that user just so happens to be a group.  I'm not ready to totally redo coldfront in that way.